roles/core/time/templates/chrony.conf.j2

```diff
@@ -3,26 +3,30 @@
 ## {{ansible_managed}}
 
 # External servers/pool if asked
-{% if external_time.time_server.server and time_profile == 'server' %}
+{% if external_time is defined and external_time.time_server is defined and time_profile == 'server' %}
```

Comment:
If we define external_time in defaults/main.yml, then we would not need to add this test.